### ISSUE TITLE
Getting "too much recursion" after upgrading to 1.10

### ISSUE THREAD

**Issue by sstratoti** [thomasloven/lovelace-auto-entities#115]
```
arch | x86_64
-- | --
dev | false
docker | true
hassio | false
installation_type | Home Assistant Core on Docker
os_name | Linux
os_version | 5.3.0-51-generic
python_version | 3.7.7
timezone | America/New_York
version | 0.110.1
virtualenv | false
```

I have this error message on my main lovelace page.
I'm using Firefox 76.0.1 (64-bit) 

```
too much recursion

card:
  attribute: battery_level
  columns: 1
  height: 25px
  positions:
    icon: 'off'
    indicator: inside
    name: outside
  severity:
    - color: '#FF0000'
      from: 1
      to: 10
    - color: '#ff6600'
      from: 11
      to: 20
    - color: '#ff9933'
      from: 21
      to: 30
    - color: '#ffcc00'
      from: 31
      to: 40
    - color: '#ffdb60'
      from: 41
      to: 50
    - color: '#ffdb60'
      from: 51
      to: 60
    - color: '#ffdb60'
      from: 61
      to: 70
    - color: '#CCFF33'
      from: 71
      to: 80
    - color: '#66FF33'
      from: 81
      to: 90
    - color: '#279b37'
      from: 91
      to: 100
  style: |-
    bar-card-value {
      margin-right: auto;
      font-size: 13px;
      font-weight: bold;
      text-shadow: 1px 1px #0005;
    }
  type: 'custom:bar-card'
  unit_of_measurement: '%'
  width: 60%
filter:
  include:
    - attributes:
        battery_level: <=30
type: 'custom:auto-entities'
```
Do I have my card set up wrong somehow? It's been working fine for a while now. Could it be an issue with the custom bar-card?

**Comment by sstratoti** [comment]
I was playing around with this trying to locate the issue with why my iOS app wasn't loading.
If I do a straight listing of entities, it works fine. When I add in the bar-card, the page in my mobile browser crashes and iOS doesn't load.
So I'm unsure if the issue lies with bar-card, or with auto-entities, but I hope I helped to narrow it down...

**Comment by dedors** [comment]
I get a "too much recursion" warning too. It affects 2 big lovelace cards with picture elements. When I hit reload it works again.

It appears to come up here when the page loads in the background, e.g. when I start my browser with multiple tabs. I can also reproduce it when I hit reload and immediately switch to another tab. This happens with firefox 77.

I did that same test (reload and switch tab) with chrome (ver 81) and the same happens, but this time the warning is "Maximum call stack size exceeded". That led me to https://github.com/ofekashery/vertical-stack-in-card/issues/90 where I figured out that it's caused by the "vertical-stack-in-card" that somehow got lost in HACS and didn't get updated.

You may need to check if your auto-entities is outdated. Remember to "shift-reload" after updating. 

_To complete the test (and if someone is googling), Edge (44.19041) ALWAYS shows "Invalid calling object". Edge does not seem able to completely load the page in the background.
(Besides that, Edge seems to fail to load custom elements. )_

**Comment by sstratoti** [comment]
hmm..I have vertical-stack-in-card loaded, but its version 3.3. And I don't actually think I'm using it anywhere. Are you saying that I may have been using an out of date auto-entities that had a similar issue?

**Comment by dedors** [comment]
Yes, I think it may be the auto-entities card that is outdated in your case, just giving the same results.

**Comment by sstratoti** [comment]
Finally was able to delve deeper into this. I actually was trying to figure out how to use the new-to-me sorting functionality of auto-entities, and it wasn't working. Then I recalled your reply, and took a look to see what resources were loading. Turns out I was loading quite a few older js files from when HACS made the jump to their own community directory. I was loading a much older version of auto-entities first, then the newer version. Undoubtedly this was causing me issues. I've removed it all and cleared my cache and everything seems to be working on the newer version. I'm going to close this because it seems to be working fine now. Thanks for pointing me in the right direction!
